chromadb/api/async_fastapi.py

```diff
@@ -465,7 +465,7 @@ async def _update(
         documents: Optional[Documents] = None,
         uris: Optional[URIs] = None,
     ) -> bool:
-        batch = (ids, embeddings, metadatas, documents, uris)
+        batch = (ids, [embedding.tolist() for embedding in embeddings] if embeddings is not None else None, metadatas, documents, uris)
```

Comment:
Same comment

Reply:
`embeddings` can be `None` here—both given the method definition and if you trace through the codepath. In `_validate_and_prepare_update_request`, we call `_validate_embedding_set` with `require_embeddings_or_data` set to `False`, which means we can end up sending `self._client._update` a `None` for `embeddings`.